test/lib/fixed_length/parser_test.rb

```diff
@@ -0,0 +1,73 @@
+require 'test_helper'
+
+class ParserTest < ActiveSupport::TestCase
+  prepend LoadBankFixture
```

Comment:
You're using it so that when you call `setup` the `@bank_file` variable gets initialized before anything happens with `@ach_format`...?